test/junit/scala/runtime/ScalaRunTimeTest.scala

```diff
@@ -5,70 +5,10 @@ import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
 
-/** Tests for the private class DefaultPromise */
+/** Tests for the runtime object ScalaRunTime */
 @RunWith(classOf[JUnit4])
 class ScalaRunTimeTest {
   @Test
-  def testIsTuple() {
-    import ScalaRunTime.isTuple
-    def check(v: Any) = {
-      assertTrue(v.toString, isTuple(v))
-    }
-
-    val s = ""
-    check(Tuple1(s))
-    check((s, s))
-    check((s, s, s))
-    check((s, s, s, s))
-    check((s, s, s, s, s))
-    check((s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-    check((s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s, s))
-
-    // some specialized variants will have mangled classnames
-    check(Tuple1(0))
-    check((0, 0))
-    check((0, 0, 0))
-    check((0, 0, 0, 0))
-    check((0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-    check((0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
-
-    case class C()
-    val c = new C()
-    assertFalse(c.toString, isTuple(c))
-  }
-
-  @Test
   def testStingOf() {
```

Comment:
Only if you feel the call. I couldn't pass up a unit test pun. Are there French scouts? or only Legionnaires? My daughter just researched the origins of Girl Scouts as a cosmopolitan effort. I was just moved to go put on my "je suis scala.js" t-shirt.